### PR TITLE
BDTopo and Common iProcess works

### PR DESCRIPTION
**Pull request by gpetit**
Improve tests and other so that BDTopo and Common iProcess works
Thanks @j3r3m1

### CODE REVIEW

File: prepare_data/src/main/groovy/org/orbisgis/bdtopo/BDTopoGISLayers.groovy [hunk 1]
```diff
@@ -64,18 +64,18 @@ static IProcess importPreprocess(){
                 logger.info('Executing the importPreprocess.sql script')
                 def uuid = UUID.randomUUID().toString().replaceAll('-', '_')
                 def tmpIris = 'TMP_IRIS_' + uuid
-                def zone = 'ZONE_' + uuid
+                def zone = 'ZONE'
```
Comment (ebocher) [inline prepare_data/src/main/groovy/org/orbisgis/bdtopo/BDTopoGISLayers.groovy]: Why delete the uuid value?

File: prepare_data/src/test/groovy/org/orbisgis/common/AbstractTablesInitializationTest.groovy [hunk 2]
```diff
@@ -24,10 +23,11 @@ class AbstractTablesInitializationTest {
     @Test
     @EnabledIfSystemProperty(named = "test.bdtopo", matches = "true")
     void initParametersAbstract(){
-        H2GIS h2GISDatabase = H2GIS.open("./target/myh2gisbdtopodb")
+        H2GIS h2GISDatabase = H2GIS.open(bdTopoDb-".mv.db", "sa", "")
         def process = PrepareData.AbstractTablesInitialization.initParametersAbstract()
         assertTrue process.execute([datasource: h2GISDatabase])
         process.getResults().each {entry ->
+            println(entry)
```
Comment (ebocher) [inline prepare_data/src/test/groovy/org/orbisgis/common/AbstractTablesInitializationTest.groovy]: Delete println

File: prepare_data/src/test/groovy/org/orbisgis/common/InputDataFormattingTest.groovy [hunk 3]
```diff
@@ -23,21 +23,38 @@ class InputDataFormattingTest {
...
+                         hLevMin: 3, hLevMax: 15, hThresholdLev2: 10, idZone: '56195',
 
                          buildingAbstractUseType: 'BUILDING_ABSTRACT_USE_TYPE', buildingAbstractParameters: 'BUILDING_ABSTRACT_PARAMETERS',
                          roadAbstractType: 'ROAD_ABSTRACT_TYPE', roadAbstractParameters: 'ROAD_ABSTRACT_PARAMETERS',
                          railAbstractType: 'RAIL_ABSTRACT_TYPE',
                          vegetAbstractType: 'VEGET_ABSTRACT_TYPE', vegetAbstractParameters: 'VEGET_ABSTRACT_PARAMETERS'])
         process.getResults().each {
-            entry -> assertNull h2GISDatabase.getTable(entry.getValue())
+            entry ->
+            println(entry)
```
Comment (ebocher) [inline prepare_data/src/test/groovy/org/orbisgis/common/InputDataFormattingTest.groovy]: idem

File: prepare_data/src/test/groovy/org/orbisgis/common/InputDataFormattingTest.groovy [hunk 4]
```diff
@@ -23,21 +23,38 @@ class InputDataFormattingTest {
     @Test
     @EnabledIfSystemProperty(named = "test.bdtopo", matches = "true")
     void inputDataFormatting(){
```
Comment (ebocher) [inline prepare_data/src/test/groovy/org/orbisgis/common/InputDataFormattingTest.groovy]: The test is a good way but we must test also if there are some rows in the formatted tables

File: prepare_data/src/test/groovy/org/orbisgis/common/AbstractTablesInitializationTest.groovy (hunk 2 above)
Comment (gpetit) [reply prepare_data/src/test/groovy/org/orbisgis/common/AbstractTablesInitializationTest.groovy]: Yes good catch. It's done in the following commit ;-)

File: prepare_data/src/test/groovy/org/orbisgis/common/InputDataFormattingTest.groovy (hunk 3 above)
Comment (gpetit) [reply prepare_data/src/test/groovy/org/orbisgis/common/InputDataFormattingTest.groovy]: Same. Removed ;-)

File: prepare_data/src/test/groovy/org/orbisgis/common/InputDataFormattingTest.groovy (hunk 4 above)
Comment (gpetit) [reply prepare_data/src/test/groovy/org/orbisgis/common/InputDataFormattingTest.groovy]: Yes, it's in the pipe. 
We are currently working to finalize all the cables and to check if tables are well created.
Then we will check if they are empty.

File: prepare_data/src/main/groovy/org/orbisgis/bdtopo/BDTopoGISLayers.groovy (hunk 1 above)
Comment (j3r3m1) [reply prepare_data/src/main/groovy/org/orbisgis/bdtopo/BDTopoGISLayers.groovy]: The uuid was sometimes used also for the output Tables while it is only useful for intermediate tables.

**Pull request merged**